step-definitions/profile/eve-payments.js

```diff
@@ -0,0 +1,84 @@
+import { client } from 'nightwatch-api';
+import { Given, When, Then } from 'cucumber';
+
+import { setupEvan } from '../../test-utils/test-utils.js';
+
+When(/I go to EVE payments tab/, async () => {
+  const evan = setupEvan(client);
+
+  await client.url(`${ evan.baseUrl }#/dashboard.evan/profile.evan/buy-eve`);
+  await client.waitForElementPresent('.buy-eve', 10 * 1000);
+});
+
+When('I select the country {string}', async (country) => {
+  client.useXpath();
+
+  const countryDropdown = '//ion-label/*[normalize-space(text()) = \'Country *\']/parent::*/following-sibling::ion-select'
+  await client.expect.element(countryDropdown).to.be.visible;
+  await client.click(countryDropdown);
+
+  await client.waitForElementPresent('//h2[normalize-space(text()) = \'Country *\']', 20e3);
+  client.pause(1e3);
+  
+  const selectedCountry = `//button[@ion-button='alert-radio-button']/*/*[normalize-space(text()) = '${country}']/parent::*/parent::*`;
+  const radioButton = await client.element('xpath', selectedCountry);
+
+  await client.execute(function (country) {
+    const radioButtons = Array.from(document.querySelectorAll('ion-alert button.alert-radio-button'));
```

Comment:
Not sure if this works reliable with css selector since you activated xpath selectors before. Better make sure to be always in the correct selection mode using `client.useCss();` / `client.useXpath();`

Reply:
Because he is in the "client.execute" scope, this is executed on the browser, instead of the script